### PR TITLE
Update nydusd doc for bootstrap creation

### PR DESCRIPTION
**Pull request by liubogithub**
Add a link for nydus-image.md as a reference.

Signed-off-by: Liu Bo <bo.liu@linux.alibaba.com>

### CODE REVIEW

**Comment by yqleng1987** [comment]
@liubogithub , a new test job has been submitted. Please wait in patience.

**Comment by yqleng1987** [comment]
@liubogithub , The CI test is completed, please check result:
<table>
    <tr>
        <th colspan="2">Test Case</th><th>Test Result</th>
    </tr>
    <tr>
        <td colspan="2">merge-target-branch</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/7284>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">build-docker-image</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/7284>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">compile-nydus</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/7284>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">compile-ctr-remote</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/7284>:x:FAIL</a></td>
    </tr>
</table>

Sorry, your test job failed. Please get the details in the link.

File: docs/nydusd.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ cat /path/to/config-localfs.json
 ```
 
 ### Run With FUSE
+If no `path/to/bootstrap` is available, please refer to [nydus-image.md](https://github.com/dragonflyoss/image-service/blob/master/docs/nydus-image.md) for more details.
```
Comment (imeoer) [inline docs/nydusd.md:27]: `/path/to/bootstrap`

File: docs/nydusd.md [hunk 2]
```diff
@@ -34,6 +35,7 @@ sudo nydusd \
 ```
 
 ### Run With Virtio-FS
+If no `path/to/bootstrap` is available, please refer to [nydus-image.md](https://github.com/dragonflyoss/image-service/blob/master/docs/nydus-image.md) for more details.
```
Comment (imeoer) [inline docs/nydusd.md:38]: Ditto.

**Comment by yqleng1987** [comment]
@liubogithub , your pull request has been updated. A new test job will be submitted. Please wait in patience.

**Comment by yqleng1987** [comment]
@liubogithub , the test job has been submitted. Please wait in patience.

**Review by imeoer — APPROVED** [review]

**Pull request merged**